changelogs/unreleased/8672-compiler-bugfix-unknown-implement-condition.yml

```diff
@@ -0,0 +1,6 @@
+description: 'Compiler: fixed bug where implement conditions evaluated Unknown values as true, deviating from how they are evaluated in if statement conditions'
+change-type: patch
+sections:
+  bugfix: "{{description}}"
+destination-branches:
+  - master
```

Comment:
Should this go into iso8? It's a bugfix, but I'm hesitant because it is a small change of behavior. I'm inclined to leave it iso9 only.